components/template-settings-json.tpl

```diff
@@ -0,0 +1,9 @@
+{
+  "site": {
+    "product_settings": {
+      "key": "PREFIX_product",
+      "value": {}
+    }
+  },
+  "version": "anchorage-2.5.2"
```

Comment:
The current version in branch is `2.5.1`. The version must be bumped only right before release.

```suggestion
  "version": "anchorage-2.5.1"
```